support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -163,28 +167,86 @@ function withProps(props: PropTypes) {
     formatAmount(currencies[props.currency], spokenCurrencies[props.currency], { value: max.toString() }, false);
   const otherAmount = props.otherAmounts[props.contributionType].amount;
 
-  return (
-    <fieldset className={classNameWithModifiers('form__radio-group', ['pills', 'contribution-amount'])}>
-      <legend className={classNameWithModifiers('form__legend', ['radio-group'])}>How much would you like to give?</legend>
-      <ul className="form__radio-group-list">
-        {validAmounts.map(renderAmount(
+  const renderContribTypeChoiceCards = (shape: string) => {
+    const sharedCss = {
+      display: 'flex',
+      justifyContent: 'flex-start',
+      border: 0,
+    };
+    const circleCss = {
+      label: {
+        borderRadius: '50%',
+        minWidth: '60px',
+        maxWidth: '60px',
+        minHeight: '60px',
+        maxHeight: '60px',
+        padding: '0',
+      },
+    };
+    const cssObj: Object = shape === 'circles' ? {
+      ...sharedCss,
+      ...circleCss,
+    } : { ...sharedCss };
```

Comment:
I don't think we need to spread the object in a new object literal, we can just pass in the object in
```suggestion
    } : sharedCss;
```